Containerfile

```diff
@@ -14,10 +14,14 @@ ENV VARIANT="Kinoite"
 ENV VERSION="${VERSION}"
 ENV WEB_UI="false"
 
-COPY / /isogenerator
+COPY ./lorax_templates /isogenerator/lorax_templates
+COPY ./xorriso /isogenerator/xorriso
+COPY ./Makefile /isogenerator/
```

Comment:
You will need to add ./scripts now that it is a dependency for secure boot.

Reply:
[302a52c](https://github.com/ublue-os/isogenerator/pull/26/commits/302a52c100ebfd848129311da3d6ef58aa48dcf9)